models/release.go

```diff
@@ -244,6 +245,19 @@ func GetReleasesByRepoID(repoID int64, page, pageSize int) (rels []*Release, err
 	return rels, err
 }
 
+// GetReleaseCountByRepoID returns the count of releases of repository
+func GetReleaseCountByRepoID(repoID int64, isOwner bool) (total int64, err error) {
```

Comment:
1. I think a variable name like `includeDrafts` would be more clear than `isOwner`.
2. Are return variable names necessary here? They don't seem to ever be used.